envoy/api/v2/lds.proto

```diff
@@ -150,4 +150,19 @@ message Listener {
   // (default), the socket is not modified, i.e. the option is neither enabled
   // nor disabled.
   google.protobuf.BoolValue freebind = 11;
+
+  // Whether the listener should accept TCP Fast Open (TFO) connections.
+  // When this flag is set to a value greater than 0, the option TCP_FASTOPEN is enabled on
+  // the socket, with a queue length of the specified size
+  // (see https://tools.ietf.org/html/rfc7413#section-5.1).
+  // When this flag is set to 0, the option TCP_FASTOPEN is disabled on the socket.
+  // When this flag is not set (default), the socket is not modified,
+  // i.e. the option is neither enabled nor disabled.
+  //
+  // On linux, the net.ipv4.tcp_fastopen kernel parameter must include flag 0x2 to enable
```

Comment:
Nit: Linux and MacOS below, thanks.

Reply:
macOS is supposed to have a lower-case m.